### PR TITLE
Add neptune logging

### LINKED ISSUE #19: Logging tool

### PR DESCRIPTION
**Pull request by jfpio**

### CODE REVIEW

File: eval_metrics.py [hunk 1]
```diff
@@ -59,6 +69,10 @@ def main(model_name: str, dataset_name: str):
     results = eval_model(model, dataset)
     print(f"metrics for {model.name}: ", results)
 
+    run["parameters"] = {"model_name": model_name}
```
Comment (dmkwis) [inline eval_metrics.py]: please use model.name instead of  model_name

**Comment by dmkwis** [comment]
Okay, please write a short section in README on how to use Neptune

**Review by dmkwis — COMMENTED** [review]
Overall okay, please write a short section in README on how to use Neptune

File: eval_metrics.py (hunk 1 above)
Comment (jfpio) [reply eval_metrics.py]: Ok

File: eval_metrics.py [hunk 2]
```diff
@@ -59,6 +69,11 @@ def main(model_name: str, dataset_name: str):
     results = eval_model(model, dataset)
     print(f"metrics for {model.name}: ", results)
 
+    run["dataset_name"] = dataset_name
+    run["parameters"] = {"model_name": model.name} # TODO There should be parameters of the model like hyperparameters
```
Comment (dmkwis) [inline eval_metrics.py:73]: Will create an issue about it and assign it to you, good point.

**Review by dmkwis — APPROVED** [review]

**Pull request merged**